tox.ini

```diff
@@ -8,14 +8,29 @@ setenv =
 
 deps =
     django111: Django>=1.11,<2.0
-    -rtest_requirements.txt
+    -r{toxinidir}/requirements/test.txt
 
 commands =
     coverage run manage.py test --verbosity=3
     coverage report -m
     coverage html
 
 [testenv:quality]
+whitelist_externals =
+    make
+deps =
+    -r{toxinidir}/requirements/quality.txt
+commands =
+    pycodestyle --config=.pep8 manage.py milestones setup.py
+    pylint --rcfile=.pylintrc manage.py milestones settings.py setup.py
+    pylint --py3k --rcfile=.pylintrc manage.py milestones settings.py setup.py
+    # isort --check-only --diff --recursive manage.py milestones settings.py setup.py
```

Comment:
Why are we commenting this out? Not ready to run it?

Reply:
Fixing the things it complained about would have added another 20 or so modified files to the PR, so I figured that should be its own separate PR.